requirements.txt

```diff
@@ -1,6 +1,11 @@
-hepunits
-numpy>=1.11.0
-pandas<0.21; python_version=="3.4.*"
-pandas; python_version == "2.7.*"
-pandas; python_version >= "3.5"
+hepunits==1.0.*
+particle==0.6.*
+decaylanguage==0.5.*
+awkward>=0.12.0,<1.0
+uproot-methods>=0.7.0,<1.0
+uproot>=3.10.0,<4.0
+boost-histogram==0.5.*
+iminuit==1.3.*
+numpy>=1.13.1
```

Comment:
This gets added by the other packages, but is fine to list here too.

Reply:
You mean numpy? I do think it makes sense to have this one here just because so many org packages use it (see my PyHEP 2019 talk) and we want to make it clear what's needed. Seems we all 3 agree, anyway.

Reply:
Yes. As an org we support 1.13.1+ only, so I think it is good here.